pants.toml

```diff
@@ -27,16 +34,20 @@ root_patterns = [
   "/dj-cmd/py",
   "/twoops_tracker/docker",
   "/twoops_tracker/py",
-  "/twoops_tracker/sh",
   "/pants-plugins",
 ]
 
 [python]
-interpreter_constraints = ["==3.9.*"]
+interpreter_constraints = ["==3.11.*"]
```

Comment:
@kilemensi just thinking out loud, do we have to restrict this to `3.11` alone? We currently have python `3.12` and would force installation of 3.11 on a new computer(local setup).
Maybe `>=3.11.*` instead?

Reply:
No we don't. **BUT**:

1. If we add other version, we'd need to add testing, etc., to make sure the code works with all supported versions i.e. it's _easier_ to target 1 version than 2 or 3.
2. We can upgrade it to `3.12` and see everything works. My only frame of reference is TrustList and there some dependencies were not ready for `3.12`.